test/format/config.h

```diff
@@ -224,6 +224,9 @@ static CONFIG c[] = {
   /* 10% */
   {"ops.alter", "configure table alterations", C_BOOL, 10, 0, 0, &g.c_alter, NULL},
 
+  /* 100% */
+  {"ops.import", "import table", C_BOOL, 100, 0, 0, &g.c_import, NULL},
```

Comment:
Are we sure we want to do this 100% of the time?

Reply:
yeah we probably don't. Not entirely sure on what % we want to do this. @sueloverso do you have a preference?

Reply:
fair, will run pull request stress testing to make sure everything is fine, then will change this back to 20%, will also run patch build to make sure there isn't anything missing.